Added a guide for JWK Verification and a repo installation troubleshooting step for windows users

wtravO opened the pull request.

benjie commented:
Awesome, thanks for this 👏 I’m going to do a few small edits tomorrow, but this is well written and informative - thanks 👍

wtravO commented:
Not a problem.  Glad to contribute. 👍

benjie commented:
I've edited your article and it's now live here:

https://www.graphile.org/postgraphile/jwk-verification/

I simplified the code samples down (removing things like `dynamicJson`, etc that were not pertinent) and I've added code highlighting. I've also replaced the `return true` / `return false` with throws to prevent someone partially following the instructions and thinking they're golden. Do have a read through and make sure I've not broken it!

wtravO commented:
Looks good to me!

On Fri, Nov 2, 2018 at 7:10 AM Benjie Gillam <notifications@github.com>
wrote:

> I've edited your article and it's now live here:
>
> https://www.graphile.org/postgraphile/jwk-verification/
>
> I simplified the code samples down (removing things like dynamicJson, etc
> that were not pertinent) and I've added code highlighting. I've also
> replaced the return true / return false with throws to prevent someone
> partially following the instructions and thinking they're golden. Do have a
> read through and make sure I've not broken it!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/graphile/graphile.github.io/pull/88#issuecomment-435345774>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJeqIjivLcpPQrCWiVJB8_6UiIgSnOVJks5urCgXgaJpZM4YIdaO>
> .
>


-- 
My regards,
Travis Oneal
